app/test_main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+def test_can_access_google_page_with_valid_url(
+        mocked_url: mock.MagicMock
+) -> None:
+    can_access_google_page("https://www.google.com/")
```

Comment:
The test currently only checks if `valid_google_url` is called, but it should also verify the return value of `can_access_google_page` to ensure it behaves as expected when the URL is valid.